terraform/testcases/jaeger_mock/parameters.tfvars

```diff
@@ -1,6 +1,6 @@
 validation_config = "jaeger-mocked-server-validation.yml"
 
-sample_app_image = "aottestbed/aws-otel-jz-sample-app:v0.1.4"
+sample_app_image = "ghcr.io/aws-observability/aws-otel-collector/jaeger-zipkin-sample-app:v0.1.5"
```

Comment:
Should this be in `aws-otel-test-framework`? The sample app isn't really a part of the collector.

Reply:
Use old image until we can make ghrc public